src/refactor_nrepl/find/find_symbol.clj

```diff
@@ -24,12 +24,12 @@
                       (str/replace "#'" "")
                       (str/replace "clojure.core/" "")))
         full-class (get alias-info class class)]
-    (str/join "/" (remove nil? [full-class (:field node)]))))
+    (str/join "/" (remove nil? [(if (map? full-class) "" full-class) (:field node)]))))
 
 (defn- contains-var?
   "Checks if the var of `node` is present in the `var-set`."
-  [vars-set alias-info node]
-  (vars-set (node->var alias-info node)))
+  [var-name alias-info node]
+  (= var-name (node->var alias-info node)))
```

Comment:
looks like `node->var` returns a single string and the `var-set` iself is a one-element set created in `contains-var-or-const?` as `#{var-name}`. single comparison should be enough here, I guess?